utils/create_levure_app_files/stack.livecodescript

```diff
@@ -524,7 +539,7 @@ on fileSetPermissions pFile, pOwner, pGroup, pAll
   put _escapeForShell(pFile) into pFile
 
   switch the platform
-    case "Win32"
+  case "Win32"
```

Comment:
@macMikey These white space changes shouldn't be part of this PR. Will you please remove them?